BMS Workspace-git/bms-1227-fw/main.c

```diff
@@ -96,6 +98,13 @@ int main(void)
        InitializeTemperature();
        setupThermistor();
 
+        if (PINMUX_PIN_54_MIBSPI3NCS_5 == 1) { // Pin 17 on X1 connector (MIBSPI3_NCS_5) is used to indicate charging mode
```

Comment:
Change pin name to something more descriptive like CHARGER_ENABLE_PIN in hwConfig.h

Reply:
done